Common/models/News.swift

```diff
@@ -29,28 +29,62 @@ class News: NSObject {
         return r.first
     }
     
-    override init() {
+    class func decodeArrayOfNews(from data: Data) -> [News] {
+        // Decoding Data into News
+        // (replace this with Decodable snippet in the future)
+        let json = try? JSON(data: data)
+        
+        var news = [News]()
+        for (_, v) in json! {
+            if let n = News(json: v) {
+                news.append(n)
+            }
+        }
+        return news
+    }
+    
+    init(identifier: String, url: URL) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/40450f80b333e8bde9095515a83e4ae5b7a64bd0/Common/models/News.swift#L46"></span>Limit vertical whitespace to a single empty line. Currently 2.</td>
    </tr>
  </tbody>
</table>